tardis/apps/publication_workflow/components/PublicationButton.jsx

```diff
@@ -0,0 +1,31 @@
+import PropTypes from 'prop-types';
+import React, { Fragment } from 'react';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933947)
```suggestion
import React, { Fragment } from "react";
```